### PR TITLE
Update 'student' to 'learner' in Insights guide text

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3474](https://openedx.atlassian.net/browse/DOC-3474)

This PR changes instances of "student" to "learner" in Insights guide. The PR updates all text and a first set of screen shots. The remaining screen shots that contain the word "student" will be updated soon.

### Date Needed (optional)
15 December 2016

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] @catong 
- [x] @stroilova 

### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: docs/en_us/dashboard/source/enrollment/Demographics_Age.rst [hunk 1]
```diff
@@ -4,89 +4,74 @@
...
+This statistic indicates that half of the learners in your course who reported
 their ages are younger, and half older, than the value that is shown.
 
 ======================================
 Age Band Metrics
 ======================================
 
 Three age ranges, or bands, are provided to give you a different at-a-glance
-perspective of the distribution of student ages. The percentage of students  in
+perspective of the distribution of learner ages. The percentage of learners  in
```
Comment (catong) [inline docs/en_us/dashboard/source/enrollment/Demographics_Age.rst]: Is there an extra space before "in"?

File: docs/en_us/dashboard/source/enrollment/Demographics_Education.rst [hunk 2]
```diff
@@ -111,25 +100,25 @@ Not Enough Calculus
...
 was too general to use as the basis for any last-minute decisions about the
 marketing or design of the course. Just in case, the professor did compile a
 list of resources for a course update.
 
 For a future course run, this professor could add an assessment early in the
-first week to test for the expected knowledge. To find the number of students
+first week to test for the expected knowledge. To find the number of learners
 who select each answer for a problem, including both incorrect and correct
-answers, you can download the `Student Answer Distribution`_ report.
+answers, you can download the `Learner Answer Distribution`_ report.
```
Comment (catong) [inline docs/en_us/dashboard/source/enrollment/Demographics_Education.rst:122]: Check whether anything else references this changed anchor/label?

File: docs/en_us/dashboard/source/performance/Performance_Answers.rst [hunk 3]
```diff
@@ -266,38 +266,38 @@ computations, see :ref:`Reference`.
...
-numerical input problem. Most students left the answer for this question
-blank, or "(empty)", which was marked incorrect. Other answers that students
+numerical input problem. Most learners left the answer for this question
+blank, or "(empty)", which was marked incorrect. Other answers that learners
 submitted, both correct and incorrect, are also graphed. The
 Submission Counts report includes a row for every submitted answer.
 
-.. image:: ../images/answer_dist_hard.png
-   :alt: A bar chart showing that most students submitted incorrect answers
+.. image:: ../images/answer_dist_hard.png`
```
Comment (catong) [inline docs/en_us/dashboard/source/performance/Performance_Answers.rst]: A stray apostrophe has been introduced after "png"

**Review by catong — CHANGES_REQUESTED** [review]
Just a couple of very minor things, then 👍

File: docs/en_us/dashboard/source/enrollment/Demographics_Gender.rst [hunk 4]
```diff
@@ -12,51 +12,44 @@ balance changes over time.
...
-.. image:: ../images/gender_chart_even.png
-   :alt: A bar chart showing 46% female and 53% male.
+.. image:: ../images/gender_chart.png
+   :alt: A bar chart showing 35.6% female, 63.5% male, and 0.9% other.
+   :width: 600
 
-.. RiceX/AdvBIOx/2014T3/enrollment/demographics/gender/
+This chart is for a science course, and learners' median age is 25. The course
+team might use the percentages of enrolled men and women as a starting point
+for an investigation into how learners learn about their course offering and
```
Comment (stroilova) [inline docs/en_us/dashboard/source/enrollment/Demographics_Gender.rst]: how learners find out about their course offering?  cc @srpearce

**Review by stroilova — APPROVED** [review]
Minor comment inline. 

Also, looks like  the change log was deleted - was that intentional?

Modulo that 👍

**Comment by srpearce** [comment]
Yep—change log was removed per a separate request. We haven't maintained change logs for about a year now, so there was no reason to keep this one.

**Pull request merged**